bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/commonChannels.xml

```diff
@@ -157,6 +157,21 @@
 	<channel-type id="humidity">
 		<item-type>Number</item-type>
 		<label>Humidity</label>
+		<state pattern="%.1f" readOnly="true" />
+	</channel-type>
+	<channel-type id="pm25">
+		<item-type>Number</item-type>
+		<label>PM2.5</label>
```

Comment:
I know what PM 2.5 probably is, but please provide a description.

Reply:
I've added a description in the latest commit